PhysicsTools/SelectorUtils/interface/CutApplicatorBase.h

```diff
@@ -33,72 +33,66 @@ namespace reco {
   typedef edm::Ptr<reco::Photon> PhotonPtr;
   typedef edm::Ptr<reco::Muon> MuonPtr;
   typedef edm::Ptr<reco::PFTau> PFTauPtr;
-}
+}  // namespace reco
 
 namespace pat {
   typedef edm::Ptr<pat::Electron> ElectronPtr;
   typedef edm::Ptr<pat::Photon> PhotonPtr;
   typedef edm::Ptr<pat::Muon> MuonPtr;
   typedef edm::Ptr<pat::Tau> TauPtr;
-}
+}  // namespace pat
 
 class CutApplicatorBase : public candf::CandidateCut {
- public:
-  enum CandidateType{NONE,
-		     ELECTRON,MUON,PHOTON,TAU,
-		     PATELECTRON,PATMUON,PATPHOTON,PATTAU};
+public:
+  enum CandidateType { NONE, ELECTRON, MUON, PHOTON, TAU, PATELECTRON, PATMUON, PATPHOTON, PATTAU };
 
- CutApplicatorBase(): CandidateCut() {}
+  CutApplicatorBase() : CandidateCut() {}
 
- CutApplicatorBase(const edm::ParameterSet& c) :
-  _name(c.getParameter<std::string>("cutName")) {
-  }
+  CutApplicatorBase(const edm::ParameterSet& c) : _name(c.getParameter<std::string>("cutName")) {}
 #if !defined(__CINT__) && !defined(__MAKECINT__) && !defined(__REFLEX__)
   CutApplicatorBase(const CutApplicatorBase&) = delete;
   CutApplicatorBase& operator=(const CutApplicatorBase&) = delete;
 #endif
-    
-  
-  result_type operator()(const argument_type&) const override
+
+  result_type operator()(const argument_type&) const
```

Comment:
Why ```override``` got lost?
Please fix

Reply:
clang-tidy removed it.

```
/pool/condor/dir_9015/jenkins/workspace/GitHub-refactor-cmssw-module/CMSSW_11_0_X_2019-06-04-2300/src/PhysicsTools/SelectorUtils/interface/CutApplicatorBase.h:62:15: warning: 'override' is redundant since the function is already declared 'final' [modernize-use-override]
  result_type operator()(const argument_type&) const override
              ^                                      ~~~~~~~~
Suppressed 1984 warnings (1975 in non-user code, 8 NOLINT, 1 with check filters)
```
I will check why clang compiler did not see the `final` keyword